platform/ui/src/components/AdvancedToolbox/ToolSettings.tsx

```diff
@@ -103,7 +103,7 @@ const renderDoubleRangeSetting = option => {
       key={option.id}
     >
       <InputDoubleRange
-        values={option.value}
+        values={option.values}
```

Comment:
this was correct, i intentionally made it value https://github.com/OHIF/Viewers/pull/4063#issue-2259722444